### PR TITLE
fixing typos in readme

### PR DESCRIPTION
**Pull request by mikemorency**
##### SUMMARY
Fixing a few typos in the README

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by anna-savina** [comment]
The [VMware REST modules guide](https://docs.ansible.com/ansible/devel/collections/vmware/vmware_rest/docsite/guide_vmware_rest.html) link doesn't work. Could you check it out please?

**Comment by mikemorency** [comment]
where do you see/how did you find that link? I think its documentation that was previously removed

File: README.md [hunk 1]
```diff
@@ -62,15 +62,72 @@ ansible-galaxy collection install vmware.vmware_rest:1.0.0
...
+
+* Use Case Name: Manage a VMs Settings
+  * Actors:
+    * System Admin
+  * Description:
+    * The system administrator can manage a VMs settings.
+  * Modules:
+    * `vmware.vmware_rest.vcenter_vm_guest_customization` - Applies a customization specification on the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_guest_filesystem_directories` - Creates a directory in the guest operating system
+    * `vmware.vmware_rest.vcenter_vm_guest_power` - Modify a virtual machine's power state
```
Comment (anna-savina) [inline README.md:107]:
```suggestion
    * `vmware.vmware_rest.vcenter_vm_guest_power` - Modifies a virtual machine's power state
```

File: README.md [hunk 2]
```diff
@@ -62,15 +62,72 @@ ansible-galaxy collection install vmware.vmware_rest:1.0.0
...
+    * `vmware.vmware_rest.vcenter_vm_hardware_boot` - Updates the boot-related settings of a virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_cdrom` - Adds a virtual CD-ROM device to the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_cpu` - Updates the CPU-related settings of a virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_disk` - Adds a virtual disk to the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_ethernet` - Adds a virtual Ethernet adapter to the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_memory` - Updates the memory-related settings of a virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_parallel` - Adds a virtual parallel port to the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware` - Updates the virtual hardware settings of a virtual machine
+    * `vmware.vmware_rest.vcenter_vm_hardware_serial` - Adds a virtual serial port to the virtual machine
+    * `vmware.vmware_rest.vcenter_vm_power` - Operate a boot, hard shutdown, hard reset or hard suspend on a guest
```
Comment (anna-savina) [inline README.md:120]:
```suggestion
    * `vmware.vmware_rest.vcenter_vm_power` - Operates a boot, hard shutdown, hard reset or hard suspend on a guest
```

File: README.md [hunk 3]
```diff
@@ -62,15 +62,72 @@ ansible-galaxy collection install vmware.vmware_rest:1.0.0
...
+    * `vmware.vmware_rest.vcenter_vm_storage_policy` - Updates the storage policy configuration of a virtual machine and/or its associated virtual hard disks
+    * `vmware.vmware_rest.vcenter_vm_tools_installer` - Connects the VMware Tools CD installer as a CD-ROM for the guest operating system
+    * `vmware.vmware_rest.vcenter_vm_tools` - Update the properties of VMware Tools
```
Comment (anna-savina) [inline README.md:123]:
```suggestion
    * `vmware.vmware_rest.vcenter_vm_tools` - Updates the properties of VMware Tools
```

**Review by anna-savina — APPROVED** [review]
LGTM

**Review by bardielle — APPROVED** [review]

**Pull request merged**